pokemongo_bot/api_wrapper.py

```diff
@@ -63,6 +64,7 @@ def call(self, max_retry=5):
             if not self._is_response_valid(result, request_callers):
                 try_cnt += 1
                 logger.log('Server seems to be busy or offline - try again - {}/{}'.format(try_cnt, max_retry), 'red')
+                logger.log('Attempted to call: {}'.format(", ".join([RequestType.Name(isinstance(req_method, dict) and req_method.keys()[0] or req_method) for req_method in api_req_method_list])), 'red')
```

Comment:
You could have used `request_callers` which basically is what you are trying to log (example: `["GET_GYM_DETAILS", "GET_INVENTORY"]`